### PR TITLE
Several changes to improve exported datasets generated by forsys

### PR DESCRIPTION
**Pull request by george-silva**
* Changes how we export / save data to ProjectArea in rscripts/forsys.R
* Changes how we export the shapefile, to use ProjectAreas, rather than direct JSON files

Now the exported project areas in shapefile matches the planning area polygon:

before:
image

after:
image

The validation STILL fails, but now it must something else.

### CODE REVIEW

File: src/planscape/planning/models.py [hunk 1]
```diff
@@ -335,7 +348,9 @@ class ProjectArea(
 
     name = models.CharField(max_length=128, help_text="Name of the Project Area.")
 
-    data = models.JSONField(null=True, help_text="Project Area data from Forsys.")
+    data = models.JSONField(
+        null=True, help_text="Project Area data from Forsys.", encoder=DjangoJSONEncoder
+    )
```
Comment (roquebetioljr) [inline src/planscape/planning/models.py:353]: I didn't find any reference to the migration. Was that done in another PR?

**Review by roquebetioljr — APPROVED** [review]

**Review by lastminutediorama — APPROVED** [review]
lgtm
